.pre-commit-config.yaml

```diff
@@ -18,7 +18,7 @@ repos:
     rev: 21.5b1
     hooks:
     -   id: black
-        language_version: python3.8
+        language_version: python
```

Comment:
huh, I didn't even know that was an option :), if it  works it works...

Reply:
had some problems if you have another python version, this works fine for me (so I think its OK)

Reply:
I think it works, let's leave it